src/main/java/sirius/biz/protocol/JournalData.java

```diff
@@ -129,18 +130,45 @@ public static void addJournalEntry(BaseEntity<?> entity, String changes) {
         addJournalEntry(entity, changes, false);
     }
 
-    private static void addJournalEntry(BaseEntity<?> entity, String changes, boolean batchLog) {
-        if (!Sirius.isFrameworkEnabled(Protocols.FRAMEWORK_PROTOCOLS) || entity.isNew()) {
+    /**
+     * Adds an entry to the journal of the given entity.
+     *
+     * @param targetId   the id of the entity under which the entity will be written
+     * @param targetType the type of the entity under which the entity will be written
+     * @param targetName the name identifying the entity, which can differ from the owner entity
+     * @param changes    the entry to add to the journal
+     */
+    public static void addJournalEntry(String targetId, String targetType, String targetName, String changes) {
```

Comment:
s.a.

Reply:
also, should we skip targetName here or force it to be "external" or kill it entirely. At this point there is no way or properly filling this in a consistent manner

Reply:
Good question. Currently the targetName ain't used anywhere. For this scenario it could be helpful in the future, especially if make it part of the searchable fields or turn it to be the type+id of the changed entity, so one could search entries based on the id of the child entity.